cluster-autoscaler/utils/drain/drain.go

```diff
@@ -224,3 +233,18 @@ func HasLocalStorage(pod *apiv1.Pod) bool {
 func isLocalVolume(volume *apiv1.Volume) bool {
 	return volume.HostPath != nil || volume.EmptyDir != nil
 }
+
+func checkPDBs(pod *apiv1.Pod, pdbs []*policyv1.PodDisruptionBudget) (bool, error) {
+	for _, pdb := range pdbs {
+		selector, err := metav1.LabelSelectorAsSelector(pdb.Spec.Selector)
+		if err != nil {
+			// "false" here is meaningless, but we can't use nil for type bool
```

Comment:
I think we can skip this comment. false is a zero value for bool and we do that in every method in CA that returns bool.